modules/default/calendar/calendar.js

```diff
@@ -40,7 +40,6 @@ Module.register("calendar", {
 		hideTime: false,
 		showTimeToday: false,
 		colored: false,
-		coloredSymbolOnly: false,
```

Comment:
I dont like removing config options without a) telling the users beforehand and b) try to provide an autoamtic replacement in code, see https://github.com/MichMich/MagicMirror/blob/4b478a5a5e891ea493171b7867ee473a15cb86cc/modules/default/weather/weather.js#L80 for an example